src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -8,8 +8,9 @@
 use Illuminate\Http\Request;
 use Illuminate\Session\SessionManager;
 use Illuminate\Session\SessionInterface;
-use Symfony\Component\HttpFoundation\Cookie;
 use Illuminate\Session\CookieSessionHandler;
+use Illuminate\Session\CacheBasedSessionHandler;
```

Comment:
Like this ? (see latest commit)
Couldn't find any coding style documentation regarding this. Wouldn't it be better to sort via namespace? I mean I don't really care but it would be great if you would document those CS guidelines somewhere. Or if StyleCI would automatically check for this.